test/test_jit.py

```diff
@@ -1621,7 +1625,8 @@ def addmm(mat, mat1, mat2, alpha, beta):
         self.run_pass('canonicalize_ops', addmm.graph)
         out_test = addmm(mat, mat1, mat2, alpha, beta)
         self.assertEqual(out_ref, out_test)
-        self.assertExpected(canonical(addmm.graph))
+        # first two addmm replaced
```

Comment:
Might be better to separate test_decompose_addmm into two tests: one for the cases that decompose, and one for the cases that don't decompose. This way we can avoid the comment and the checking logic is clearer